src/common/components/search-input/search.css

```diff
@@ -0,0 +1,10 @@
+$border-color: $mid-grey;
+
+.selectRepositorySearch {
+  border: 1px solid $border-color;
+  border-radius: 2px;
+  margin-left: 1rem;
+  outline: none;
+  padding: .275rem .75rem;
+  vertical-align: baseline;
+}
```

Comment:
There is still the 'search' icon missing from the input, like on designs from #715.

Maybe that one will do: https://assets.ubuntu.com/v1/4a72426e-search-icon-grey.svg?